WonSeungHyun/app.js

```diff
@@ -37,3 +35,118 @@ const start = async () => {
 };
 
 start();
+
+app.post("/users", async (req, res, next) => {
+  const { name, email, profile_image } = req.body;
+
+  await appDataSource.query(
+    `
+    INSERT INTO users(
+    name,
+    email,
+    profile_image
+    
+  )VALUES (?, ?, ?)`,
+    [name, email, profile_image]
+  );
+  res.status(201).json({ message: "successfully created" });
+});
```

Comment:
해당 부분도 마찬가지입니다.

매개변수로 작성한 next는 불필요해보입니다. 해당 api는 json 형식으로 특정 메시지를 반환하기 때문입니다.

추가로 sql 쿼리문 포매팅도 고려해서 다시 작성해주세요! 가독성을 높이는 방법 중 한가지입니다.

```suggestion
app.post("/users", async (req, res) => {
  const { name, email, profile_image } = req.body;
  
  await appDataSource.query(
    `
    INSERT INTO users(
      name,
      email,
      profile_image
    ) VALUES (?, ?, ?)`,
      [name, email, profile_image]
  );
  res.status(201).json({ message: "successfully created" });
});
```